internal/integ/cluster/utils.go

```diff
@@ -75,40 +75,50 @@ func getCredentialsFromEnv() (string, string, error) {
 	return apiKey, apiSecret, nil
 }
 
-func (c *Cluster) getClusterID() (string, error) {
+func (c *Cluster) getClusterID() (exov3.UUID, error) {
 	if err := flags.ValidateFlags(); err != nil {
 		return "", err
 	}
 
-	cluster, err := c.Ego.FindSKSCluster(c.exoV2Context, *flags.Zone, *flags.ClusterName)
+	clusterList, err := c.Ego.ListSKSClusters(c.context)
```

Comment:
clusterList should have a method find, or you need to bump Egoscale to latest :)

Reply:
Just checked, latest version of egoscale have it
```Golang
FindSKSCluster(nameOrID string) (v3.SKSCluster, error)

FindSKSCluster attempts to find an SKSCluster by name or ID.
```